buildconfig/pygame-stubs/mask.pyi

```diff
@@ -40,7 +40,7 @@ class Mask:
     ) -> None: ...
     def copy(self) -> Mask: ...
     def get_size(self) -> Tuple[int, int]: ...
-    def get_rect(self, **kwargs) -> Rect: ...  # Dict type needs to be completed
+    def get_rect(self, **kwargs: Any) -> Rect: ...  # Dict type needs to be completed
```

Comment:
I wonder if the comment makes sense now? Has the dict type been completed here?

Reply:
From my reading of the C code, this code really can accept anything and is a helper to set basically any property of this class.

Potentially the comment has been removed, but I'm uncertain that this typing can do much better unless someone takes the time to unravel all possible args and then turns them into named arguments. I'm certainly no expert there.

Adding `Any` makes the type not unknown, which makes some type checkers (like pyright) unhappy. We did this to typeshed a bit ago and that's kept clean. Whatever works here, I just narrowed these stubs down to as few issues as possible (the only problems remaining are the presence of default values, but I'd rather not remove them if they are being kept up to date; a new PEP outlines that default values are acceptable in some cases and they are useful for editors).

Reply:
> the only problems remaining are the presence of default values

This is because we get some extra information because of that default values (more in this comment: https://github.com/pygame/pygame/pull/2050#pullrequestreview-483058627 (3rd point)). But if it is problem for some tools maybe we can consider removing them

Reply:
Yes, I totally understand, which is why I left them. Usually it's a bad idea because you have to ensure they are properly synced, but I have no problem leaving them and figuring that out later. Like I said, a new PEP about stub conventions may change this up.